python/tvm/relay/op/transform.py

```diff
@@ -1322,6 +1322,71 @@ def adv_index(inputs):
     return _make.adv_index(Tuple(inputs))
 
 
+def sparse_fill_empty_rows(sparse_indices, sparse_values, dense_shape, default_value):
+    """
+    Fill first column of the empty rows with default values for a sparse array.
+    It returns a TupleWrapper with 3 outputs
+    Parameters
+    ----------
+    sparse_indices : relay.Expr
+        A 2-D int64 tensor[N, ndims] of integers containing location of sparse values, where N is
+        the number of sparse values and n_dim is the number of dimensions of the dense_shape.
+        The inputs for this relay parameter must be in row-major order.
```

Comment:
Row major order means that the first dimension of the array indicates the row indices. I think what you want to say here is that `sparse_indices` must be sorted such that the first column is in ascending order.

Reply:
Yes. Thank you. Done.